pkg/cache/v2/simple.go

```diff
@@ -366,8 +378,65 @@ func createResponse(request *Request, resources map[string]types.ResourceWithTtl
 	}
 }
 
+// CreateDeltaWatch returns a watch for a delta xDS request which implements the Simple SnapshotCache.
```

Comment:
Ah yea, i'll take off the part `which implements the Simple SnapshotCache`. Then it will read like the SOTW comment. Unless it's worth while differentiating the two here since the request cycle is a little different in delta